src/prefs/ApplicationPrefs.cpp

```diff
@@ -68,12 +69,44 @@ void ApplicationPrefs::PopulateOrExchange(ShuttleGui & S)
    S.SetBorder(2);
    S.StartScroller();
 
-   S.StartStatic(XO("Update Audacity"));
+   /* i18n-hint: Title for the update notifications panel in the preferences dialog. */
+   S.StartStatic(XO("Update notifications"));
    {
-      S.TieCheckBox(
-          XO("&Check for Updates...").Stripped(TranslatableString::Ellipses | TranslatableString::MenuCodes),
-          DefaultUpdatesCheckingFlag);
+      /* i18n-hint: Check-box title that configures periodic updates checking. */
+      S.TieCheckBox(XC("Check for updates", "application preferences"), DefaultUpdatesCheckingFlag);
```

Comment:
Ask David whether this or other strings should have ampersands.  If so, make sure no letter is used twice on the page.

Reply:
Should the default state in that BoolSetting be changed?

Reply:
> Ask David whether this or other strings should have ampersands. If so, make sure no letter is used twice on the page.

Will do.

Reply:
David says, that it will make sense for the checkbox, the link is good without it

Reply:
Sometimes he wants them for static text boxes too?  His call.